website/docs/getting-started/flashing-nrf52.md

```diff
@@ -33,6 +65,9 @@ Be careful to install the correct load for your board. While it is unlikely that
 
 * Connect your device to your computer with a USB cable. If you computer complains about needing to format a new drive, cancel the format command.
 * Double click the reset button on your device, this will put it into boot loader mode.
+
+[<img alt="LILYGO T-Echo" src="/img/hardware/t-echo-lilygo.jpg" style={{zoom:'25%'}}  />](/img/hardware/t-echo-lilygo.jpg)
+
 * A new drive will then be mounted on your computer. Open this drive and you should see three files: `CURRENT.UF2`, `INDEX.HTM`, and `INFO_UF2.TXT`
-* Copy the appropriate `firmware-xxxxx-1.2.x.uf2` file from the firmware zip file onto the new drive.
+* Copy the appropriate `firmware-xxxxx-1.2.x.uf2` file from the firmware zip file onto the new drive. NOTE: You are going to copy/drop "as is" 'firmware-xxxxx-1.2.x.uf2' (NOT over "CURRENT.UF2") in the volume, and the device reboot will copy it/load it correctly.
```

Comment:
Another place for an admonition.

Reply:
Done - added (overwrote latest push - see new file)